www/%team/index.html.spt

```diff
@@ -37,7 +37,7 @@ title = name = team.name
 <div class="col0">
     {% if team.is_approved in (None, False) %}
     {% if team.is_approved == None %}unreviewed{% endif %}
-    {% if team.is_approved == False %}rejected{% endif %} |
+    {% if team.is_approved == False %}closed{% endif %} |
```

Comment:
Good find! I looked for that in the database schema, I didn't think to check for a computed property.